SQLitePCL.pretty/SQLiteException.cs

```diff
@@ -76,17 +76,14 @@ internal static Exception Create(ErrorCode rc, ErrorCode extended, string msg)
         /// </summary>
         public ErrorCode ExtendedErrorCode { get; }
 
-        private readonly string errmsg;
-
-        private SQLiteException(ErrorCode errorCode, ErrorCode extendedErrorCode, string msg)
+        private SQLiteException(ErrorCode errorCode, ErrorCode extendedErrorCode, string msg) : base(msg)
         {
             this.ErrorCode = errorCode;
             this.ExtendedErrorCode = extendedErrorCode;
-            errmsg = msg;
         }
 
         /// <inheritdoc/>
         public override string ToString() =>
-            $"{ErrorCode}: {errmsg}\r\n{base.ToString()}";
+            $"{ErrorCode}: {Message}\r\n{base.ToString()}";
```

Comment:
```suggestion
            $"{ErrorCode}: {base.ToString()}";
```
The base class already displays the message.